gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineScalastyle.java

```diff
@@ -0,0 +1,98 @@
+/*
```

Comment:
General concern - does this addition make the `gradle-baseline-java` and `gradle-baseline-java-config` jar names semantically wrong now? Should we add `gradle-baseline-scala` / `gradle-baseline-scala-config`?

Reply:
uh, yes? Usually you have both java and scala in one project anyway. Should we move BaselineIdea/BaselineEclipse out to it's own subproject as well?